### PR TITLE
feat: Add getPrivateKey method for key signer

### PR DESCRIPTION
**Pull request by 2woongjae**
The Adena wallet has a UI that shows the private key.
For this, we need a method to return the private key from `wallet.signer`.

Thanks.

### CODE REVIEW

File: src/wallet/signer.ts [hunk 1]
```diff
@@ -13,6 +13,11 @@ export interface Signer {
    */
   getPublicKey(): Promise<Uint8Array>;
 
+  /**
+   * Returns the signer's Secp256k1-compressed private key
```
Comment (zivkovicmilos) [inline src/wallet/signer.ts:17]: I would note that this is not the compressed private key, but the actual raw private key

File: src/wallet/ledger/ledger.ts [hunk 2]
```diff
@@ -44,6 +44,10 @@ export class LedgerSigner implements Signer {
     return this.connector.getPubkey(this.hdPath);
   };
 
+  getPrivateKey = async (): Promise<Uint8Array> => {
+    throw new Error('Ledger does not support private key');
```
Comment (zivkovicmilos) [inline src/wallet/ledger/ledger.ts:48]: We should expand this message to be:

`Ledger does not support private key exports`

**Review by zivkovicmilos — APPROVED** [review]
Looks great 💯 

Thank you for opening up this PR 🙏 

I've left 2 small comments, and we should be good to merge

File: src/wallet/signer.ts (hunk 1 above)
Comment (2woongjae) [reply src/wallet/signer.ts]: @zivkovicmilos
I missed something in the comments
I changed it to `actual raw private key`.

**Pull request merged**